site/source/docs/compiling/Running-html-files-with-emrun.rst

```diff
@@ -178,6 +179,8 @@ To run on Android, add the ``--android`` command line flag and use the ``--brows
 
 .. note:: Omitting ``--browser`` (to launch a default Android browser) is not supported. 
 
+.. note:: Running on Android will omit the ``--hostname`` option
```

Comment:
I just didn't want to change the logic of the original --android option here: https://github.com/juj/emrun/blob/master/emrun.py#L1339-L1340